dist/pkgbuild/PKGBUILD.in

```diff
@@ -0,0 +1,40 @@
+# Maintainer: Christopher Hoage <iam@chrishoage.com>
+
+pkgname=ipmi-fan-control
+pkgver=@VERSION@
+pkgrel=1
+pkgdesc="SuperMicro IPMI fan control daemon"
+url="https://github.com/chenxiaolong/$pkgname"
+source=("@TARBALL_NAME@")
+backup=("etc/$pkgname.toml")
+arch=("x86_64")
+license=("GPLv3+")
+makedepends=("cargo")
+depends=("ipmitool")
+optdepends=("smartmontools")
+sha256sums=(
+  "dc6fa88529fee2dacfe2b09fdeaa1e392c8a45e9821011231a60e76235a64d77"
+)
+
+build() {
+  cd "$srcdir/$pkgname-$pkgver"
+
+  cargo build --release
+}
+
+package() {
+  cd "$srcdir/$pkgname-$pkgver"
+
+  install -m755 -d "${pkgdir}/usr/lib/systemd/system/"
+
+  sed \
+      -e "s/@BINDIR@/\/usr\/bin/g" \
+      -e "s/@SYSCONFDIR@/\/etc/g" \
+      < dist/ipmi-fan-control.service \
+      > "${srcdir}"/ipmi-fan-control.service
+
+  install -Dm644 "${srcdir}"/ipmi-fan-control.service "${pkgdir}/usr/lib/systemd/system/"
```

Comment:
One small tweak from your suggestion just to ensure the file is installed with the correct permissions